python/ray/tests/test_client_reconnect.py

```diff
@@ -430,6 +430,34 @@ def fail_halfway(_):
         assert result.shape == (1024, 1024, 128)
 
 
+def test_disconnect_during_large_schedule():
+    """
+    Disconnect during a remote call with a large (multi-chunk) argument.
+    """
+    i = 0
+    started = False
+
+    def fail_halfway(_):
+        # Inject an error halfway through the object transfer
+        nonlocal i, started
+        if not started:
+            return
+        i += 1
+        if i == 8:
+            raise RuntimeError
+
+    @ray.remote
+    def f(a):
+        return a.shape
+
+    with start_middleman_server(on_data_request=fail_halfway):
+        started = True
+        a = np.random.random((1024, 1024, 128))
```

Comment:
Can you mention why this is split into several messages (e.g. that it is X MiB, and OBJECT_TRANSFER_CHUNK_SIZE == Y MiB)?